cmd/kubelet/app/options/options.go

```diff
@@ -588,8 +588,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.BoolVar(&c.ProtectKernelDefaults, "protect-kernel-defaults", c.ProtectKernelDefaults, "Default kubelet behaviour for kernel tuning. If set, kubelet errors if any of kernel tunables is different than kubelet defaults.")
 
 	// Node Allocatable Flags
-	fs.Var(flag.NewMapStringString(&c.SystemReserved), "system-reserved", "A set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=500Mi,ephemeral-storage=1Gi) pairs that describe resources reserved for non-kubernetes components. Currently only cpu and memory are supported. See http://kubernetes.io/docs/user-guide/compute-resources for more detail. [default=none]")
-	fs.Var(flag.NewMapStringString(&c.KubeReserved), "kube-reserved", "A set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=500Mi,ephemeral-storage=1Gi) pairs that describe resources reserved for kubernetes system components. Currently cpu, memory and local ephemeral storage for root file system are supported. See http://kubernetes.io/docs/user-guide/compute-resources for more detail. [default=none]")
+	fs.Var(flag.NewMapStringString(&c.SystemReserved), "system-reserved", "A set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=500Mi,ephemeral-storage=1Gi,pids=100) pairs that describe resources reserved for non-kubernetes components. Currently only cpu, memory, and pids are supported. See http://kubernetes.io/docs/user-guide/compute-resources for more detail. [default=none]")
```

Comment:
the example shows "pids", but the resource name is "process-ids".

Reply:
Good catch.  Fixed.

Reply:
Should we make this consistent with the eviction signal, which is [`pid.available`](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/api/types.go#L42).  Should we make it singular as well?  We use `cpu` instead of `cpus`, so should we use `pid`/`process-id` instead of `pids`/`process-ids`?

Reply:
Only counterpoints are:

1. It's very evident what `cpu` means.  It's less self-evident what `pid` refers to.

2. CPU can refer to fractional units.  Process IDs are always expressed as an integer.

But I don't have a really strong opinion on this, and I'm happy to go any of the 4 possible ways.

Reply:
I think I prefer being consistent with the eviction signals and using `pid`.  I wish we had used `process-ids.available` for eviction to start with, but such is life.

Reply:
Done.

I've had to pull my test out; it's failing for very non-obvious reasons having to do with the cgroups not existing.  That started after I rebooted my laptop.  However, the node container manager test is passing, so at least in that context the right cgroups are being set up.